raise_search/commands/index_content.py

```diff
@@ -0,0 +1,58 @@
+import argparse
+import csv
+from raise_search.api.utils import get_opensearch_client
+from pathlib import Path
+from bs4 import BeautifulSoup
+
+
+def index_content(index, toc_csv_path, html_directory):
+    with toc_csv_path.open() as csv_file:
+        items = csv.DictReader(csv_file)
+        for item in items:
+            index_item(item, index, html_directory)
+
+
+def index_item(item, index, html_directory):
+    html_content = Path(
+        f"{html_directory}/{item['content_id']}.html").resolve(strict=True).read_text()
+    soup = BeautifulSoup(html_content, 'html.parser')
+
+    doc = {
+        "content_id": item["content_id"],
+        "section": item["section"],
+        "activity_name": item["activity_name"],
+        "lesson_page": item["lesson_page"],
+        "lesson_page_type": item["lesson_page_type"],
+        "teacher_only": item["visible"] == "0",
+        "visible_content": soup.get_text()
+    }
+    client = get_opensearch_client()
```

Comment:
Instead of recreating the `client` on each item, we can instantiate it in `index_content` and then pass it to `index_item` so we re-use it.